docs/sources/installation/requirements.md

```diff
@@ -23,31 +25,32 @@ The following operating systems are supported for Grafana installation:
 - [macOS](/installation/mac)
 - [Windows](/installation/windows)
 
-## Unsupported operating systems
+While you can install Grafana on other operating systems, we do not recommend doing so and do not provide support for that use case.
 
-Installation of Grafana on other operating systems is possible, but not supported. Please see the [building from source](/project/building_from_source/#building-grafana-from-source) guide for more information.
+## Hardware recommendations
 
-## Hardware requirements
+Grafana does not use a lot of resources and is very lightweight in use of memory and CPU. 
 
-Grafana does not use a lot of resources and is very lightweight in use of memory and CPU. Minimum recommendation is 255mb of memory and 1 CPU.
+Minimum recommended memory: 255 MB
+Minimum recommended CPU: 1
 
-Depending on what features are being used and to what extent the requirements varies. Features that consume and requires more resources:
+Some features might require more memory or CPUs. Features require more resources include:
 
-- [Server side rendering of images](/administration/image_rendering/#requirements)
+- [Server-side rendering of images](/administration/image_rendering/#requirements)
 - [Alerting](/alerting/rules/)
-- Data source proxy
+- Data source proxies
 
-## Database
+## Supported databases
 
-Grafana requires a database to store its configuration data, e.g. users, data sources and dashboards. The exact requirements depend on the size of the Grafana installation (e.g. the number of users, data sources, dashboards, features in use etc).
+Grafana requires a database to store its configuration data, such as users, data sources, and dashboards. The exact requirements depend on the size of the Grafana installation and features used.
 
 Grafana supports the following databases:
 
```

Comment:
Should there be a link to instructions for changing databases?

Reply:
Probably a good idea. We currently only have this documentation https://grafana.com/docs/installation/configuration/#database so probably some documentation lacking there.